### DISCUSSION TITLE
client certificate

### DISCUSSION THREAD

**Discussion by pragyasandeep1998hal** [Keyfactor/ejbca-ce#710] [General]
Hello

Through Golang code I send an Initial registration request and in response I get one CA certificate and one server certificate.

I want server and client both certificates. Can u tell me is there any way or configuration so that I can get both server and client certificate. Or do I need to send separate requests to get server and client certificate separately.

**Comment by primetomas** [comment]
You don't say how you request certificates, what protocol are you using?

**Comment by pragyasandeep1998hal** [comment]
I am using CMPv2 protocol

**Comment by pragyasandeep1998hal** [comment]
I am using CMPv2 protocol

Can you tell me how we can get both client and server certificates from EJBCA server

**Comment by primetomas** [comment]
You can only make one certificate request in each CMP call, so make two calls.

**Comment by pragyasandeep1998hal** [comment]
ok thanks for the confirmation
